### ISSUE TITLE
For `GLM-10B-Chinese`, the fine-tuning loss barely decrease within each epoch and it only decreases when starting a new epoch.

### ISSUE THREAD

**Issue by silverriver** [THUDM/GLM#88]
I am using `GLM-10B-Chinese` with the following script:

```bash
bash scripts/ds_finetune_seq2seq.sh \ 
   config_tasks/model_blocklm_10B.sh \ 
   config_tasks/seq_cnndm_org.sh
```

I use a customized dataset with about 6k examples. However, the training loss steps down with the training epoches, i.e., after certain steps, the training loss barely decrease within each epoch, but there is a large decrease when starting a new epoch.

Something looks like this:

![loss image](https://i.ibb.co/xYHK4wM/20230211100551.png)

or this link if you can not see the image (https://ibb.co/PMQsSJ5)

I am wondering have you encountered this issue?

I have checked the dataset shuffling logic, the dataset is shuffled in every epoch, and the tensorboard summaries are writted correctly.

**Comment by duzx16** [comment]
This is normal. When it enters a new epoch, all the examples are trained for one more time in the previous epoch and the loss is naturally lower. The loss remains unchanged within the same epoch since all the examples have been trained for the same number of times (as the number of epochs).

**Comment by silverriver** [comment]
Thank for your reply.

**Comment by shmily326** [comment]
> Thanks for your reply.

Sorry to bother you, I'm wondering what's the initial loss of your loss curve, when fine-tuning GLM10B I got the loss curve below, but I'm not sure how to validate whether the loss is valid or rational, looking forward to your reply, thanks.
<img width="351" alt="image" src="https://github.com/THUDM/GLM/assets/24571857/d9b578ff-5224-49d2-96ef-ee73c68b494f">
